### PR TITLE
Setup guide for local development

### PR DESCRIPTION
**Pull request by styxit**
- Created a `.gitignore` file to exclude build pages.
- Added a gem file to install local dependencies.
- Wrote a guide on how to get the techblog working on local machines.

### CODE REVIEW

File: DEV_SETUP.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Setup Exonet techblog on your local machine
+The Exonet techblog is hosted at github-pages. Because the thechblog automatically rebuilds all pages after committing to the `master` branch, every commit that is pushed to the `master` branch is immediately live.
```
Comment (robinmulder) [inline DEV_SETUP.md]: techblog

File: DEV_SETUP.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+## Requirements and setup
+- **Xcode** with commandline/development tools - `xcode-select --install`
+- **Ruby** >= 2.0.0 - Use `ruby --version` to verify this.
+- **Bundler** - `gem install bundler`
+
+## Install github pages dependencies
+The server at github pages that builds and hosts the techblog does not always run the latest versions of all dependencies. There is a [Github pages dependency versions](https://pages.github.com/versions) that show the current versions of all dependencies.
+
+### Updating dependencies
+To keep your local dev environment in sync, a [github-pages ruby gem](https://rubygems.org/gems/github-pages) is used. It can be found in the `Gemfile`. This gem is updated when de dependency versions change. Run `bundle update` to check if a new version is available for this gem.
```
Comment (robinmulder) [inline DEV_SETUP.md]: the dependency

File: DEV_SETUP.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+## Running a local version of the techblog
+On the commandline run `bundle exec jekyll serve`. This command creates a tiny webserver at [127.0.0.1:4000](127.0.0.1:4000) serving the techblog with your local changes.
+
+Any time a file is modified, the pages are rebuild and the a new version is served.
```
Comment (robinmulder) [inline DEV_SETUP.md]: -a

File: DEV_SETUP.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+Most of this can be found at [Github help](https://help.github.com/articles/using-jekyll-with-pages).
+
+## Requirements and setup
+- **Xcode** with commandline/development tools - `xcode-select --install`
+- **Ruby** >= 2.0.0 - Use `ruby --version` to verify this.
+- **Bundler** - `gem install bundler`
+
+## Install github pages dependencies
+The server at github pages that builds and hosts the techblog does not always run the latest versions of all dependencies. There is a [Github pages dependency versions](https://pages.github.com/versions) that show the current versions of all dependencies.
```
Comment (robbinjanssen) [inline DEV_SETUP.md]: Suggestion:

`There is a [Github pages dependency versions](https://pages.github.com/versions) that show the current versions of all dependencies.`

-->

`Fortunately, Github provides a [page](https://pages.github.com/versions) that shows the current dependencies and versions for a Github pages blog.`

Comment (styxit) [inline DEV_SETUP.md]: I don't see any added value in the suggested change because it looks to me the message is exactly the same?

Comment (robbinjanssen) [inline DEV_SETUP.md]: For me it was unclear what `Github pages dependency versions` exactly was.

**Comment by trizz** [comment]
:+1: :shipit:

**Pull request merged**